.werft/util/shell.ts

```diff
@@ -35,6 +19,8 @@ export function exec(command: string, options: ExecOptions & { async?: false }):
 export function exec(command: string, options: ExecOptions & { async: true }): Promise<ExecResult>;
 export function exec(command: string, options: ExecOptions): shell.ShellString | ChildProcess;
 export function exec(cmd: string, options?: ExecOptions): ChildProcess | shell.ShellString | Promise<ExecResult> {
+    const werft = getGlobalWerftInstance()
+
```

Comment:
Can we create a span here, too? This would help to track the commands that are executing during the build incl. their arguments and durations. Hope this is not too redundant to the log in Werft - but in honecomb we can get the timings and in Werft we don't have them.

Reply:
(this may also be out-of-scope for this PR and could be done in a follow-up-PR)

Reply:
Currently it will produce a log slice if you provide a `slice` key in the options. So we have two options

1. Change invocations of `exec` to pass in `slice` in the options for the ones we care about
2. Change the logic so that all `exec`'s produce log slices (using the cmd as the name of the slice if a name isn't provided)

Do you have a preference? :)

Reply:
Oof, isn't it too much to add spans to every exec? 😅 

I guess we could send some to see how it looks like in honeycomb though...

Reply:
@meysholdt I agree this would be better done in a follow up PR :)

Reply:
don't we care about all execs? I think traces will be more helpful to explain failures or time-singks when there is not a single exec missing.

Also, I think spans are a bit more fine-grained than [Werft slices](https://github.com/csweichel/werft#log-cutting), because slices are mostly used to express paralelism while spans are measure durations. 

Furthermore, the logic below does not seem to log `cmd` in all cases. In the trace, we should have that in all case.

Reply:
> Oof, isn't it too much to add spans to every exec?

werft log: a beautiful on-point guided tour through the activity of a job.
trace: show me all the dirty details!

Reply:
I think it is just my spider-senses against high-cardinality tingling.

Let's try in a future iteration and see how it goes!

![image](https://user-images.githubusercontent.com/24193764/140062916-e8bed818-040a-47c4-9e31-ca96113e016d.png)